x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rule_details/components/use_rule_description_fields.tsx

```diff
@@ -128,51 +125,68 @@ export const createPrebuildFields = ({
       }),
       description: <span>{id}</span>,
     }),
-    [RULE_PREBUILD_DESCRIPTION_FIELDS.DATA_VIEW_INDEX_PATTERN]: (indexId: string) => ({
-      title: i18n.translate('xpack.triggersActionsUI.ruleDetails.dataViewIndexPatternTitle', {
-        defaultMessage: 'Data view index patterns',
-      }),
-      description: (
-        <AsyncField<
-          | {
-              result: {
-                result: { item: { id: string; attributes: { title: string; name: string } } };
-              };
-            }
-          | undefined
-        >
-          queryKey={['esQueryRuleDescriptionDataViewDetails']}
-          queryFn={() => {
-            return http!.post('/api/content_management/rpc/get', {
-              body: JSON.stringify({
-                contentTypeId: 'index-pattern',
-                id: indexId,
-                version: '1',
-              }),
-            });
-          }}
-        >
-          {(data) => {
-            if (!data) {
-              return <div data-test-subj="description-detail-data-view-pattern-error">-</div>;
-            }
-
-            try {
-              const dataViewIndexPatterns = data.result.result.item.attributes.title.split(',');
+    [RULE_PREBUILD_DESCRIPTION_FIELDS.DATA_VIEW_INDEX_PATTERN]: (dataViewIndexId: string) => {
+      const { dataViews } = useKibana().services;
+      return {
+        title: i18n.translate('xpack.triggersActionsUI.ruleDetails.dataViewIndexPatternTitle', {
+          defaultMessage: 'Data view index patterns',
+        }),
+        description: (
+          <AsyncField<DataView>
+            queryKey={[RULE_DESCRIPTION_GET_DATA_VIEW_QUERY_KEY]}
```

Comment:
we are gonna be doing the same call for the data view index pattern field and for the query filters. I'm using the same key in both to avoid calling the api twice when using both fields